src/components/EResourceSections/PCIFormInfo.js

```diff
@@ -49,6 +50,25 @@ export default class PCIFormInfo extends React.Component {
             />
           </Col>
         </Row>
+        <Row>
+          <Col xs={12}>
+            <Field
```

Comment:
If this was c/ped from my code, [it doesn't need to be this complicated](https://github.com/folio-org/ui-agreements/pull/537).

Reply:
Yeah, fixed it.